internal/log/log.go

```diff
@@ -81,41 +81,83 @@ func SetLogLevel(lvl Level) {
 }
 
 // Output format should be: "timestamp•LOG_LEVEL•filename.go•linenumber•output"
-func Fatal(format string, args ...interface{}) {
-	logger.Fatal(format, args...)
+func Fatal(msg interface{}, args ...interface{}) {
```

Comment:
I'm wondering if we should setup a second set of methods like `Fatalf`, Errorf, Warnf, etc. to match with the convention of having a message format string like the stdlib `log` package uses.

Reply:
I agree here. We should have two set's of functions, following the way the stdlib has set things up. The `f` variant should take a format string, and the non-f variant should not just take a bunch of interfaces and just pass them on with doing anything special to them.

Reply:
Are you guys sure?  I think it's a good idea that it matches the standard log interface and suggested it during the design meeting, but you didn't want it then.

Reply:
I think there may have been a misunderstanding; during that meeting. I do not recall not wanting to follow the standard log interface. If that was the case I apologize, we do want to follow the go-developers standards as much as possible — as long as it makes sense.